libs/langchain-community/langchain.config.js

```diff
@@ -128,6 +127,7 @@ export const config = {
     "vectorstores/turbopuffer": "vectorstores/turbopuffer",
     "vectorstores/typeorm": "vectorstores/typeorm",
     "vectorstores/typesense": "vectorstores/typesense",
+    "vectorstores/upstash": "vectorstores/upstash",
```

Comment:
Also need to add to `requiresOptionalDependency` since you need to install the `@upstash/vector` package.